tests/test_meteodatalab/test_grib_decoder.py

```diff
@@ -39,7 +39,7 @@ def test_save(data_dir, tmp_path):
 
 @pytest.mark.parametrize("param", ("U", "V", "T"))
 def test_save_field(data_dir, tmp_path, param):
-    datafile = data_dir / "COSMO-1E/1h/ml_sl/000/lfff00000000"
```

Comment:
It would be nice to add a tests. For example in the delta operator which changes the length of time range, to be sure now that the information about the length of time range is not overwitten.